scripts/basic-workflow-tests.sh

```diff
@@ -256,12 +257,33 @@ test_convert() {
 }
 
 test_evaluate() {
+    local MODEL_PATH="${DATA_HOME}/instructlab/models/instructlab/granite-7b-lab"
+    # Temporarily using merlinite as the base model to confirm the workflow executes correctly 
+    local BASE_MODEL_PATH="${DATA_HOME}/instructlab/models/instructlab/merlinite-7b-lab"
     task Evaluate the model
 
     if [ "$EVAL" -eq 1 ]; then
       export INSTRUCTLAB_EVAL_MMLU_MIN_TASKS=true
       export HF_DATASETS_TRUST_REMOTE_CODE=true
-      ilab model evaluate --model "${DATA_HOME}/instructlab/models/instructlab/granite-7b-lab" --benchmark mmlu
+      task Running MMLU
+      ilab model evaluate --model "${MODEL_PATH}" --benchmark mmlu
+      task Running MMLU_BRANCH
+      ilab model evaluate --model "${MODEL_PATH}" --benchmark mmlu_branch --tasks-dir tests/testdata/mmlu_branch/ --base-model "${BASE_MODEL_PATH}"
+      export INSTRUCTLAB_EVAL_FIRST_N_QUESTIONS=5
+      task Running MT_Bench
+      ilab model evaluate --model "${MODEL_PATH}" --judge-model "${MODEL_PATH}" --enable-serving-output --benchmark mt_bench
+      cd "${DATA_HOME}/instructlab/taxonomy"
+      git branch rc
```

Comment:
What's the goal of this `git branch rc` command? I am not sure it changes anything in the cloned repo files, except creating an additional `rc` branch (pointing to the previous `HEAD`). What am I missing?

```
$ git clone git@github.com:instructlab/taxonomy
Cloning into 'taxonomy'...
remote: Enumerating objects: 3422, done.
remote: Counting objects: 100% (20/20), done.
remote: Compressing objects: 100% (16/16), done.
remote: Total 3422 (delta 7), reused 15 (delta 4), pack-reused 3402
Receiving objects: 100% (3422/3422), 2.80 MiB | 9.45 MiB/s, done.
Resolving deltas: 100% (1325/1325), done.
$ cd taxonomy/
$ git log --oneline HEAD~..
d472f50 (HEAD -> main, origin/main, origin/HEAD) Added table from #786 (#1189)
$ git branch rc
$ git log --oneline HEAD~..
d472f50 (HEAD -> main, origin/main, origin/HEAD, rc) Added table from #786 (#1189)
```

Reply:
Nevermind, I think I understand why you have to create a branch (it's passed as --branch later.)

Reply:
Yes, we need to pass a taxonomy repo branch to eval QNAs against model and we're using an rc branch to do that